gremlin/ui/device.py

```diff
@@ -997,4 +1100,18 @@ def axisIdentifier(self, index: int) -> int:
         fset=_set_window_size,
         fget=_get_window_size,
         notify=windowSizeChanged
-    )
\ No newline at end of file
+    )
+
+
+Configuration().register(
+    "global",
+    "input-names",
+    "input-name-display-mode",
+    PropertyType.Selection,
+    "Mixed",
+    "Defines how input name is displayed.",
+    {
+        "valid_options": ["None", "Mixed", "Name"]
```

Comment:
"None" is a somewhat confusing label as it might be interpreted as showing no labels at all. Maybe "Numerical", "Numerical & Label", "Label" might be clearer?

Reply:
Yup, I agree.